ui/css/utilities/colors.scss

```diff
@@ -12,3 +17,4 @@
   // Required for the QR reader to work properly
   --qr-code-white-background: #fff;
 }
+/* stylelint-enable color-no-hex */
```

Comment:
Remove this line
```suggestion
```

Reply:
Updated https://github.com/MetaMask/metamask-extension/pull/23098/commits/bcec68c8bac52e2b5bf97d750b290e15b5f17d19